versioned_docs/version-5.x/faq/migrating-from-ink-4-to-5.md

```diff
@@ -403,8 +403,21 @@ The argument type changed from `u32` to `u16`:
 +Function,
 ```
 
+The top level macro now accepts (extension = N: u16) argument to support multiple chain extensions.
+If you are using only one extension, the ID can be any `u16` number, 
+otherwise please consult [`#[ink::chain_extension]` page](https://use.ink/5.x/macros-attributes/chain-extension)
```

Comment:
This will break compatibility for the scenario you described. Consulting the target chain's docs is the most sensible approach, as the chain registry repo appears to be unmaintained (the last changes were 2 years ago).

Until then, if you are sure that the chain only uses a single chain, then putting any number. is fine